homeassistant_satellite/__main__.py

```diff
@@ -235,6 +236,7 @@ async def main() -> None:
             host=args.host,
             protocol=args.protocol,
             port=args.port,
+            skipVerify=args.skipVerify,
```

Comment:
```suggestion
            skipVerify=args.skipVerify,
```

Python generally uses [snake case](https://en.wikipedia.org/wiki/Snake_case), not camel case.

Reply:
done